lib/transport.rb

```diff
@@ -0,0 +1,24 @@
+class Transport
+  include Comparable
+
+  attr_accessor :max_weight, :speed, :available
+  attr_reader :delivery_speed_by_weight
```

Comment:
you applied attr_reader for method instead of variable, it will work without it. remove it please